ironfish/src/network/peers/peerConnectionManager.ts

```diff
@@ -110,21 +110,11 @@ export class PeerConnectionManager {
   }
 
   private connectToEligiblePeers(peer: Peer): boolean {
-    if (peer.state.type !== 'CONNECTED') {
-      if (this.peerManager.canConnectToWebRTC(peer)) {
-        if (this.peerManager.connectToWebRTC(peer)) {
-          return true
-        }
-      }
-
-      if (this.peerManager.canConnectToWebSocket(peer)) {
-        if (this.peerManager.connectToWebSocket(peer)) {
-          return true
-        }
-      }
+    if (peer.state.type === 'CONNECTED') {
+      return false
     }
 
-    return false
+    return this.peerManager.connectToWebRTC(peer) || this.peerManager.connectToWebSocket(peer)
```

Comment:
I think on this I still would rather prefer 
```javascript
if(this.peerManager.connectToWebRTC(peer)) {
  return true
}

return this.peerManager.connectToWebSocket(peer)
```

Just makes it a little more explicit that an action is being taken and `connectToWebRTC(peer)` is not just a boolean

Reply:
done